### PR TITLE
add jquery.terminal v0.9.3 with autoupdate config

### LINKED ISSUE #7285: [Request] Add jquery.terminal
**Library name:** jquery.terminal 
**Git repository url:** https://github.com/jcubic/jquery.terminal
**npm package url(optional):** https://www.npmjs.com/package/jquery.terminal
**License(s):** LGPL
**Official homepage:** http://terminal.jcubic.pl/

### PR DESCRIPTION
**Pull request by dannyxx001**
@pvnr0082t please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jcubic/jquery.terminal
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/jquery.terminal
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1269
   - Count of watchers: 77
   - Count of forks: 292
 * NPM download stats (optional):
   - Downloads in the last day: 4
   - Downloads in the last week: 18
   - Downloads in the last month: 63

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7285, cc @tg123

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@dannyxx001 It's not the time to review each other's PR until we finished the first training of @cdnjs/intern2 .

**Comment by PeterDaveHello** [comment]
1. css file not minified.
2. Filename should not contain version number.

**Comment by dannyxx001** [comment]
ok, I will ask more detail and fix it tomorrow because I face some problems now.

**Comment by PeterDaveHello** [comment]
@dannyxx001 You can just ask and discuss here.

**Comment by dannyxx001** [comment]
OK, first, when I use "git checkout" to another branch, the lib directory was gone. Did I make something wrong?

**Comment by PeterDaveHello** [comment]
@dannyxx001 Have you checked your sparse-checkout config?

**Comment by PeterDaveHello** [comment]
Filename should not contain version number, you didn't fix it.

https://github.com/cdnjs/cdnjs#conventions

**Comment by dannyxx001** [comment]
@PeterDaveHello I have passed all checks, please help me review this pull request, thank you.

File: ajax/libs/jquery.terminal/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "jquery.terminal",
+  "npmName": "jquery.terminal",
+  "npmFileMap": [
+    {
+      "basePath": "js",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.terminal/package.json]: Should not use `"**/*"` in non-dist folder since you can not make sure it's for distribution.

**Comment by PeterDaveHello** [comment]
Your auto-update config will not grab css files.

**Comment by dannyxx001** [comment]
@PeterDaveHello please help me check this pull request, thank you.

File: ajax/libs/jquery.terminal/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  "homepage": "http://terminal.jcubic.pl",
+  "dependencies": {
+    "jquery": "^2.1.4"
+  },
+  "gitHead": "8f7d5e0999caf2ca75e1418e47df426ac741320f",
+  "dist": {
+    "shasum": "524968ed08b8e1eab3c600056f3d589599d2862f",
+    "tarball": "http://registry.npmjs.org/jquery.terminal/-/jquery.terminal-0.9.3.tgz"
+  }
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.terminal/package.json]: Please remove `gitHead` and `dist`

File: ajax/libs/jquery.terminal/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "jquery.terminal",
+  "npmName": "jquery.terminal",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/jquery.terminal-min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.terminal/package.json]: Please also add `jquery.terminal-src.js`

File: ajax/libs/jquery.terminal/package.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        "css/jquery.terminal.css"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.terminal/package.json]: Please also add `jquery.terminal-src.css`

**Comment by PeterDaveHello** [comment]
Please put js files into `js` folder and so does `css`, as your auto-update config does.

**Comment by dannyxx001** [comment]
@PeterDaveHello I have put them in their directory, please help me check again, thank you.

File: ajax/libs/jquery.terminal/package.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "jquery.terminal",
+  "npmName": "jquery.terminal",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/jquery.terminal-min.js",
+        "js/jquery.terminal-src.js",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.terminal/package.json]: You should try `"js/jquery.terminal-+(min|src).js",` next time.

File: ajax/libs/jquery.terminal/package.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        "css/jquery.terminal-src.css",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.terminal/package.json]: `jquery.terminal-src.css` is duplicated with `jquery.terminal.css`

Comment (dannyxx001) [inline ajax/libs/jquery.terminal/package.json]: OK, I have modified it.

File: ajax/libs/jquery.terminal/package.json (hunk 5 above)
Comment (dannyxx001) [inline ajax/libs/jquery.terminal/package.json]: OK, I have modified it.

**Comment by tg123** [comment]
Thanks @dannyxx001

**Comment by PeterDaveHello** [comment]
@dannyxx001 there is no `js/jquery.terminal.js` in that npm package, but in your commit, so this is a wrong PR that I merged in accident.

**Comment by dannyxx001** [comment]
Sorry, I made mistake. I forgot to follow the name from author.

**Pull request merged**